### ISSUE TITLE
对于MMTLB的问题

### ISSUE THREAD

**Issue by plyon1** [rzhao-zhsq/CV-SLT#6]
您知道MMTLB中Sign2Gloss2Text (two-stage)和Sign2Text (end-to-end)是怎么训练的吗，按照官方给出的训练步骤好像是Sign2Text，我不知道该如何区分这两种情况，这两种情况有什么区别。如果您知道的话，可以给我解释一下吗，谢谢！！

**Comment by rzhao-zhsq** [comment]
@plyon1 Hi，
MMTLB的这两个方式都要先训练S2G和G2T两个阶段,推理时将S2G的gloss结果直接作为G2T模型的输入就是S2G2T. 
S2T在这个基础上进行S2(G+T)的e2e训练。训练和推理一致，都是输入S,用S2G的输出（visual head/logits）作为为G2T模型的输入.

**Comment by plyon1** [comment]
感谢您的回复！也就是我先按照步骤先训练S2G和G2T两个阶段，那我如何得到S2G的gloss结果并将其输入到G2T模型中，是不是执行 prediction.py的时候config文件给${dataset}_g2t.yaml，但是这样的话我没有在这个文件中看到S2G的gloss结果配置路径。是要像${dataset}_s2t.yaml中一样，在${dataset}_g2t.yaml加入head_rgb_input的路径吗？

**Comment by rzhao-zhsq** [comment]
是的，官方代码没有S2G2T的pipeline，你要自行推理得到S2G的结果，然后将得到的结果组织输入给G2T模型。注意G2T的输入是gloss序列，而不是head_rgb_input特征，head_rgb_input特征是用于S2(G+T)相关的训练。由于直接输入视频显存需求较高，所以MMTLB是提取了S2G的特征进行的S2(G+T)的e2e训练。

**Comment by plyon1** [comment]
所以，其实本质上来说MMTLB的Sign2Text也是基于gloss，并不是现在广泛研究的gloss-free任务。我明白了，这个问题困扰我很久，万分感谢您的及时回复！

**Comment by rzhao-zhsq** [comment]
是的，很高兴能帮到你！
